scripts/qa_crowbarsetup.sh

```diff
@@ -1275,7 +1276,26 @@ function onadmin_setup_local_zypper_repositories
     # restore needed repos depending on localreposdir_target
     if [ -n "${localreposdir_target}" ]; then
         mount_localreposdir_target
-        uri_base="file:///repositories"
+        if [ -z "$localreposdir_is_clouddata" ]; then
+            uri_base="file:///repositories"
+        else
+            localrepos_base="file://$localreposdir_target/repos/$arch"
+            case $(getcloudver) in
+                6)
+                    $zypper ar "$localrepos_base/SLES12-SP1-Pool" sles12sp1
+                    $zypper ar "$localrepos_base/SLES12-SP1-Updates" sles12sp1up
+                ;;
+                7)
+                    $zypper ar "$localrepos_base/SLES12-SP2-Pool" sles12sp2
+                    $zypper ar "$localrepos_base/SLES12-SP2-Updates" sles12sp2up
+                ;;
+                8)
+                    $zypper ar "$localrepos_base/SLES12-SP3-Pool" sles12sp3
+                    $zypper ar "$localrepos_base/SLES12-SP3-Updates" sles12sp3up
```

Comment:
I am afraid we can't do that. Those paths differ so much. I believe the next step, to factor out the lookup will deal with that.